e2e/auth.setup.ts

```diff
@@ -34,12 +34,6 @@ for (const {
     })
 
     setup('create and authenticate user', async () => {
-      // This may be the first test to run and trigger SWC compilation
-      // ...which may take 15+ seconds. Mark this test as slow,
-      // ...so the test will be given triple timeout.
-      // https://playwright.dev/docs/api/class-test#test-slow-1
-      setup.slow()
```

Comment:
Successful `auth.setup` tests are taking 2-4 seconds on the larger Linux runner, 4-6 seconds locally, and 7-11 seconds on the macOS runner. Generally, this test is much faster than the other e2e tests, so stop marking this test as "slow" -- allow the test to fail faster when flaky, so we spend spend less money on larger runner in CI.